_posts/2019-06-06-NeuralODEs.markdown

```diff
@@ -0,0 +1,388 @@
+
+  **Topics**:
+
+  1. Initial values problems
+  2. One-step methods
+  3. Consistency and convergence
+
+  **Required Reading**:
+
+  1. Sections 12.1-4 from (Süli & Mayers)
+  2. Sections 11.1-3 from (Quarteroni et al.)
+  
+  **Notes**: Here is a [link](/assets/nodes_notes/week1.pdf) to our notes for the lesson.
+  
+  **Optional Reading**:
+
+  1. Runge-Kutta methods: Section 12.5 from (Süli & Mayers)
+  2. [Prof. Trefethen's class ODEs and Nonlinear Dynamics 4.2](http://podcasts.ox.ac.uk/odes-and-nonlinear-dynamics-42)
+
+  **Questions**:
+
+  1. Exercise 1 in Section 11.12 of (Quarteroni et al.)
+     <details><summary>Solution</summary>
+     <p>
+     Start from the hint in the book and notice that you can bound \(E_2\) as
+     
+     $$|E_2|\leq \frac{hL}{2}|y_{n+1}-y_{n} - hf(t_n,y_n)| = \frac{hL}{2}O(h^2) = O(h^3)$$
+     
+     where \(L\) is the Lipschitz constant of \(f\). On the other hand, \(E_1 = O(h**3)\) (see [link](https://en.wikipedia.org/wiki/Trapezoidal_rule#Error_analysis)).
+     </p>
+     </details>
+
+  2. Exercises 12.3,12.4, 12.7 in Section 12 of (Süli & Mayers)
+     <details><summary>Solutions</summary>
+     <p>
+     Exercise 12.3: 
+     Notice that we can write
+     
+     $$(y + q/p)'=p(y + q/p)$$
+     
+     It follows that \(y(t) = Ce^{pt} - q/p\) for some constant \(C\). Imposing the initial condition \(y(0)=1\), we get \(y(t)=e^{pt} + q/p(e^{pt}-1)\). In particular, we expand \(y\) in its Taylor series : \(y(t) = 1 + (1+q/p)\sum_{k=1}^\infty \frac{(pt)^k}{k!}\). To conclude the exercise we only need to notice that \(y_n(t) = q/p + (1+q/p)\sum_{k=1}^n \frac{(pt)^k}{k!}\) satisfies Picard's iteration: \(y_0 \equiv 1\), \(y_{n+1}(t) = y_0 + \int_0^t (py_n(s) + q)\,ds\).
+     </p>
+     <p>
+     Exercise 12.4:
+     Applying Euler's method with step-size \(h\), we get \(\hat{y}(0) = 0\), \(\hat{y}(h) = \hat{y}(0) + h \hat{y}(0)^{1/5} = 0\), \(\hat{y}(2h) = \hat{y}(h) + h \hat{y}(h)^{1/5} =0\). Iterating, we see that it holds \(y(nh)=0\) for all \(n\geq 0\). Instead, the implicit Euler's method reads
+     
+     $$\hat{y}_{n+1} = \hat{y}_n + h \hat{y}_{n+1}^{1/5}$$
+     
+     for \(n \geq 0\) and \(\hat{y}_0 = 0\). After substituting \(\hat{y}_{n} = (C_nh)^{5/4}\) in the above relation, we only need that there exists a sequence \(C_n\) satisying the requirements.
+     </p>
+     <p>
+     Exercise 12.7: 
+     First, notice that
+     
+     $$e_{n+1} = y(x_{n+1}) - y_{n} - \frac{1}{2}h(f_{n+1} + f_n)= e_n - \frac{1}{2}h (f_{n+1}+f_n) + \int_{x_n}^{x_{n+1}} f(s,y(s))\,ds$$
+     
+     and that the second component of the RHS is the same as \(E_1\) in Exercise 1 above. Therefore the first bound follows. The last inequality is simply obtained by re-arranging the terms.  
+     </p>
+     </details>
+
+  3. Consider the following method for solving \(y' = f(y)\):
```

Comment:
In “method for solving (y’ = f(y))”, remove the parentheses.

Reply:
does not that show up as latex?

Reply:
that showed up fine. however, there is another issue below it:
"(0 \leq\theta\leq 1)" is not showing up fine.